main.js

```diff
@@ -1,6 +1,42 @@
-import './style.css'
-
-document.querySelector('#app').innerHTML = `
-  <h1>Hello Vite!</h1>
-  <a href="https://vitejs.dev/guide/features.html" target="_blank">Documentation</a>
-`
+const APIURL = "https://api.themoviedb.org/3/discover/movie?sort_by=popularity.desc&api_key=04c35731a5ee918f014970082a0088b1&page=1";
+const IMGPATH = "https://image.tmdb.org/t/p/w1280";
+const SEARCHAPI = "https://api.themoviedb.org/3/search/movie?&api_key=04c35731a5ee918f014970082a0088b1&query=";
+const movieGrid = document.querySelector("#movie-grid")
+const getMovies = async (api) => {
+    const response = await fetch(api)
+    const data = await response.json()
+    let display_size = data.results.slice(0,15)
+    showMovies(display_size)
+}
+
+const showMovies = (data) => {
+    movieGrid.innerHTML = "";
+    data.forEach(
+        (item) => {
+            const movieGridItem = document.createElement("div")
+            movieGridItem.classList.add("movie-grid-item")
+            movieGridItem.innerHTML = `

```

Comment:
Using innerHTML affects the performance and can be a security issue, so please refactor this part to not use innerHTML.